PLUGIN_DEV.md

```diff
@@ -1,20 +1,19 @@
-# Plugin Development 
+# Plugin development 
 
 This document is not meant as a complete guide for developing plugins but more as a changelog for changes in
 Grafana that can impact plugin development. Whenever you as a plugin author encounter an issue with your plugin after
 upgrading Grafana please check here before creating an issue. 
 
-## Links
+## Plugin development resources
 
-- [Datasource plugin written in TypeScript](https://github.com/grafana/typescript-template-datasource)
-- [Simple JSON datasource plugin](https://github.com/grafana/simple-json-datasource)
-- [Plugin development guide](http://docs.grafana.org/plugins/developing/development/)
+- [Grafana plugin developer guide](http://docs.grafana.org/plugins/developing/development/)
 - [Webpack Grafana plugin template project](https://github.com/CorpGlory/grafana-plugin-template-webpack)
+- [Simple JSON datasource plugin](https://github.com/grafana/simple-json-datasource)
```

Comment:
We could also fix the datasource occurrence, i.e. `[Simple JSON data source plugin]`.

Reply:
I would rather not. The source document is entitled "Simple JSON datasource." Generally, we want to match the target title. I left the "plugin" so that there would be some clue as to what was on the other end of the link, since otherwise the title is terribly vague.